src/ethereum/frontier/vm/instructions/system.py

```diff
@@ -69,6 +70,10 @@ def create(evm: Evm) -> None:
         evm.message.current_target,
         get_account(evm.env.state, evm.message.current_target).nonce - U256(1),
     )
+    is_collision = account_has_code_or_nonce(evm.env.state, contract_address)
+    if is_collision:
+        push(evm.stack, U256(0))
+        return None
```

Comment:
```suggestion
        return
```

This should be fine, I think?

Reply:
Done